### ISSUE TITLE
Could not invoke Instabug.logUserEventWithNameAndParams

### ISSUE THREAD

**Issue by SudoPlz** [Instabug/instabug-reactnative#143]
Hey there,

we recently received the following crash from one of our devices in production:

```
java.lang.RuntimeException Could not invoke Networking.sendRequest 
    JavaMethodWrapper.java:385 com.facebook.react.bridge.JavaMethodWrapper.invoke
    JavaModuleWrapper.java:162 com.facebook.react.bridge.JavaModuleWrapper.invoke
    NativeRunnable.java:-2 com.facebook.react.bridge.queue.NativeRunnable.run
    Handler.java:789 android.os.Handler.handleCallback
    Handler.java:98 android.os.Handler.dispatchMessage
    MessageQueueThreadHandler.java:31 com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage
    Looper.java:164 android.os.Looper.loop
    MessageQueueThreadImpl.java:194 com.facebook.react.bridge.queue.MessageQueueThreadImpl$3.run
    Thread.java:764 java.lang.Thread.run


Caused by: java.lang.reflect.InvocationTargetException
    Method.java:-2 java.lang.reflect.Method.invoke
    JavaMethodWrapper.java:374 com.facebook.react.bridge.JavaMethodWrapper.invoke
    JavaModuleWrapper.java:162 com.facebook.react.bridge.JavaModuleWrapper.invoke
    NativeRunnable.java:-2 com.facebook.react.bridge.queue.NativeRunnable.run
    Handler.java:789 android.os.Handler.handleCallback
    Handler.java:98 android.os.Handler.dispatchMessage
    MessageQueueThreadHandler.java:31 com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage
    Looper.java:164 android.os.Looper.loop
    MessageQueueThreadImpl.java:194 com.facebook.react.bridge.queue.MessageQueueThreadImpl$3.run
    Thread.java:764 java.lang.Thread.run


Caused by: java.lang.OutOfMemoryError pthread_create (1040KB stack) failed: Try again 
    Thread.java:-2 java.lang.Thread.nativeCreate
    Thread.java:733 java.lang.Thread.start
    ThreadPoolExecutor.java:970 java.util.concurrent.ThreadPoolExecutor.addWorker
    ThreadPoolExecutor.java:1388 java.util.concurrent.ThreadPoolExecutor.execute
    Dispatcher.java:129 okhttp3.Dispatcher.enqueue
    RealCall.java:82 okhttp3.RealCall.enqueue
    NetworkingModule.java:331 com.facebook.react.modules.network.NetworkingModule.sendRequest
    Method.java:-2 java.lang.reflect.Method.invoke
    JavaMethodWrapper.java:374 com.facebook.react.bridge.JavaMethodWrapper.invoke
    JavaModuleWrapper.java:162 com.facebook.react.bridge.JavaModuleWrapper.invoke
    NativeRunnable.java:-2 com.facebook.react.bridge.queue.NativeRunnable.run
    Handler.java:789 android.os.Handler.handleCallback
    Handler.java:98 android.os.Handler.dispatchMessage
    MessageQueueThreadHandler.java:31 com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage
    Looper.java:164 android.os.Looper.loop
    MessageQueueThreadImpl.java:194 com.facebook.react.bridge.queue.MessageQueueThreadImpl$3.run
    Thread.java:764 java.lang.Thread.run
```

and here are a couple of breadcrumbs right before the issue:

<img src="https://cl.ly/1B451i0y2l3f/Image%202018-04-16%20at%207.10.12%20PM.png" />

"instabug-reactnative": "2.8.0",
"react-native": "0.53.3",
Device: Physical, Android 8.0.0 API 26 Samsung Galaxy Note 8 N950U

**Comment by Korazy** [comment]
Hey @SudoPlz 

Is this issue still occurring for you with the latest release?

**Comment by SudoPlz** [comment]
Hey @Korazy I'll have to check to be sure, I have no idea atm..

**Comment by SudoPlz** [comment]
@Korazy this is still happening on instabug-reactnative 2.10.0, on which version is it fixed?

**Comment by Korazy** [comment]
@SudoPlz I'm not sure the exact version number but it was definitely later than 2.10.0. Fix should be around version 2.13.
